content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -79,6 +79,7 @@ different Kubernetes components.
 | `CSIMigrationOpenStackComplete` | `false` | Alpha | 1.17 | |
 | `CSIMigrationvSphere` | `false` | Beta | 1.19 | |
 | `CSIMigrationvSphereComplete` | `false` | Beta | 1.19 | |
+| `CSIServiceAccountToken` | `false` | Alpha | 1.20 | |
```

Comment:
Looks to me this new gate is not documented?
We need a short description for each feature gate on this page. Please see the last section.

Reply:
@zshihang looks like I missed this before merging, could you open up another PR for description?

Reply:
https://github.com/kubernetes/website/pull/25028